file-formats/intf/intf.json

```diff
@@ -195,46 +195,46 @@
       }
     },
     "methods": {
-      "type": "array",
       "title": "Method Descriptions",
-      "description": "Method descriptions of the interface",
+      "description": "Method descriptions",
+      "type": "array",
       "items": {
+        "title": "Method Description",
+        "description": "Method description",
         "type": "object",
-        "title": "Method",
-        "description": "Method with name, description, parameter descriptions and exception descriptions",
         "properties": {
           "name": {
-            "type": "string",
-            "maxLength": 30,
             "title": "Method Name",
-            "description": "Name of the method"
+            "description": "Name of the method",
+            "type": "string",
+            "maxLength": 30
           },
           "description": {
-            "type": "string",
-            "maxLength": 60,
             "title": "Method Description",
-            "description": "Description of the method"
+            "description": "Description of the method",
+            "type": "string",
+            "maxLength": 60
           },
           "parameters": {
-            "type": "array",
             "title": "Parameter Descriptions",
-            "description": "Parameter descriptions of the method",
+            "description": "Parameter descriptions",
+            "type": "array",
             "items": {
+              "title": "Component Description",
+              "description": "Component description (e.g., for methods, attributes, types)",
```

Comment:
Here, we were more specific before. Before the title was `Parameter`, now it is `Component Description`.